src/components/FeaturedPhotoSelector.jsx

```diff
@@ -69,28 +79,37 @@ export default function FeaturedPhotoSelector({
           flexDirection: 'column',
         }}
       >
-        {error && (
+        {(sightingError || individualError) && (
           <CustomAlert
             style={{ marginBottom: 20 }}
             severity="error"
             titleId="ERROR_UPDATING_PROFILE"
-            description={error}
+            description={sightingId ? sightingError : individualError}
```

Comment:
```
const error = sightingError || individualError;
if (error) {
  // ...
  description={error}
}
```
This way the error that causes the custom alert to render is the one that is displayed.

Reply:
Done